### PR TITLE
fix global cli

### LINKED ISSUE #200: Global variables options in command line.
Hi, @feross ,

I know we can set options in package.json or put some comments in the head of the files.
But we still met some troubles when use some plugins in our Editors(like Atom), the cwd is no longer the root path of our project, so if we could have some options to config the global variables in the command line like the ignores, it will be perfect!!

### PR DESCRIPTION
**Pull request by ricardofbarros**
This will fix #200

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ function parseOpts (opts) {
   if (!opts.ignore) opts.ignore = []
   opts.ignore = opts.ignore.concat(DEFAULT_IGNORE)
 
-  setGlobals(opts.globals || opts.global)
+  setGlobals(opts.global)
```
Comment (feross) [inline index.js]: This change is not right. I want to support either option name when people `require('standard')` in node.js. Even if the command line uses only `global`, we should keep the other.

File: bin/cmd.js [hunk 2]
```diff
@@ -85,6 +87,7 @@ if (argv.stdin) {
       })
     }
   }
+
```
Comment (feross) [inline bin/cmd.js]: Avoid random whitespace changes next time

**Comment by feross** [comment]
Thanks! Released, with fixes as described, as 5.0.0-4.

File: bin/cmd.js (hunk 2 above)
Comment (ricardofbarros) [inline bin/cmd.js]: My bad

File: index.js (hunk 1 above)
Comment (ricardofbarros) [inline index.js]: Okay, understood :sweat_smile:

**Pull request merged**